code/style.css

```diff
@@ -1,7 +1,594 @@
+  border: solid #373B3E;
+  border-width: 0 3px 3px 0;
+  top: 5px;
+  left: 7px;
+  transform: rotate(45deg);
+  display: none;
+}
+
+
+button {
+  width: 130px;
+  position: relative;
+  border: none;
+  background-color: transparent;
+  cursor: pointer;
+  font-size: 14px;
+  text-transform: uppercase;
+  padding: 20px 15px;
+  margin: 20px 20px;
+  font-family: inherit;
+
+}
+
+button::before, button::after {
+  content:'';
+  display: block;
+  position: absolute;
+  width: 20%;
+  height: 20%;
+  border: 2px solid lightgrey;
+  transition: all .6s ease;
+  border-radius: 2px;
+}
+
+button::after {
+  bottom: 0;
+  right: 0px;
+  border-top-color: transparent;
+  border-left-color: transparent;
+}
+
+button::before {
+  top: 0;
+  left: 0;
+  border-bottom-color: transparent;
+  border-right-color: transparent;
+}
+
+button:hover:after, button:hover:before {
+  width:100%;
+  height: 100%;
+}
+
+
+/***ACCORDION***/
+.accordion-text-container.active {
+  height: 10vh;
+}
+
+/****FOOTER****/
+
+footer {
+  background-color: rgb(208, 211, 209);
+  display: grid;
+  align-content: center;
+  gap: 15px;
+}
+
+.copyright, .contacts, .social {
+  padding: 2vw;
+}
+
+.copyright img {
+  height: 70px;
+  margin-top: 10px;
+}
+
+.contacts span {
+  font-family: 'Montserrat', sans-serif;
+  font-weight: 200;
+  margin-left: 10px;
+  font-size: 18px;
+}
+
+#address, #phone {
+  margin-bottom: 15px;
+}
+
+
+.social i {
+  color:black;
+  margin: 5px;
+}
+
+/***TABLET MEDIA QUERIES****/
+@media (min-width: 767px) {
+  .nav-bar {
+    all: unset;
+    display: flex;
+    list-style: none;
+    align-self: center;
+  }
+
+  .nav-bar a {
+    font-size: 18px;
+    margin-right: 3vw;
+  }
+
+  .menu-icon {
+    display: none;
+  }
+
+  .hero {
+    flex-direction: column;
+  }
+
+  .bakery-container {
+   flex-direction: row;
+  }
+
+  .food-container {
+    flex-direction: row-reverse;
+  }
+
+  .bakery-container img, .food-container img {
+    width: 50%;
+  }
+
+  .bakeryText, .foodText {
+    width: 40%;
+    margin-left: 10px;
+  }
+
+  /***ACCORDION***/
+  .accordion-text-container.active {
+    height: 10vh;
+  }
+
+  /***FORM***/
+  .form-wrapper {
+    justify-content: flex-start;
+    margin-left: 10vw;
+  }
+
+  /*FOOTER*/
+  footer {
+    grid-template-columns: 30% 30% 30%;
+    justify-content: center;
+    padding: 3vw;
+  }
+}
+
+/****DESKTOP MEDIA QUERIES****/
+
+@media (min-width: 1024px) {
+  .nav-bar {
+    display: flex;
+  }
+
+  .nav-bar a {
+    font-size: 22px;
+  }
+
+  .menu-icon {
+    display: none;
+  }
+
+  .hero {
+    flex-direction: column;
+    align-items: flex-start;
+
+  }
+
+  .bakery-container {
+    flex-direction: row;
+  }
+ 
+   .food-container {
+     flex-direction: row-reverse;
+  }
+ 
+   .bakery-container img, .food-container img {
+     width: 35%;
+  }
+ 
+   .bakeryText, .foodText {
+     width: 40%;
+     margin-left: 10px;
+  }  
+
+  /***FOOTER***/
+
+  footer {
+    grid-template-columns: 30% 30% 30%;
```

Comment:
Another way to write this is: `grid-template-columns: repeat(3, 30%);`, where the 3 is the amount of times it should be repeated and then you specify the width. This is extra useful if you have a grid with something like 10-20 columns ✌️